_data/roadmap.yml

```diff
@@ -1,4 +1,53 @@
 future:
+  - title: Open Data Hub 1.3 - (May/June 2022)
+    version: 1.3
+    children:
+      - title: Tutorials and website cleanup
+      - title: Dashboard updates
+        description: Admin - BYON + Cluster Settings
+      - title: Bring your own notebooks (with and without Pipelines)
+        description: Allow admins the flexibility to provide to users their own notebook images
+      - title: Model Serving
+        description: Implementation of Model Mesh using Oauth
+      - title: Community governance model
+        url: https://github.com/opendatahub-io/opendatahub-community/issues/11
+      - title: Kubeflow 1.5 support
+        description: Includes ability to install Kubeflow v1.5.0 components on OpenShift
+        description: Support for the integration of Service Mesh with Kubeflow v1.5 on OpenShift
+        description: Track issues
+        url: https://github.com/opendatahub-io/manifests/projects/2
+
+  - title: Open Data Hub 1.4 - (June/July 2022)
+    version: 1.4
+    children:
+      - title: Model Serving
+        description: Updates to the etcd and minio setups
+      - title: Support for ODH Notebook Controller
+        url: https://github.com/opendatahub-io/odh-dashboard/projects/1
+      - title: Admin UI feature(s)
+        description: Add UI support for admin configurations
+      - title: ODH Re-architecture - Phase 1
+        description: Categorize existing components (as of ODH v1.3.0) into support Tiers for updates/deprecation and designate maintainers
+        description: Tiered approach
+          - Tier 0 - ODH Core
```

Comment:
This is not rendered as a list.

Reply:
If you add the html line break `<br>` to the end of each line (header & list items) then it will render in the style of a list

Reply:
@LaVLaS here is how I am resolving this.

        description: |
           Categorize existing components (as of ODH v1.3.0) into support Tiers for updates/deprecation and designate maintainers
           Tiered approach
             Tier 0 - ODH Core
             Tier 1 - Long Term community support with assigned MAINTAINERS and full testsuite
             Tier 1 - All other components